tools/integration_tests/util/setup/setup.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package setup
```

Comment:
what's the difference between setup and static_mounting packages ? seems like setup is now helper for only_dir_mounting and static_mounting. if yes, why it shouldn't have RunTests function ?

Reply:
Yes setup becomes helper for only_dir_mounting and static_mounting as well other mounting also which we will define in future. setup has RunTests for common tests like with mountedDirectory flag. Also at the end mounting from different ways we are calling ExecuteTest function from setup.

Reply:
if setup is to be used for mounting then do you think we can organize the code something like:
- util
   - mounting
      - setup (you can name it better, something like mounting/common)
      - static
      - only-dir

Reply:
fixed